apps/src/templates/studioHomepages/MarketingAnnouncementBanner.jsx

```diff
@@ -94,6 +95,15 @@ const MarketingAnnouncementBanner = ({announcement, marginBottom}) => {
   // and the value of displayBanner may change.
   const bannerDisplayStyle = displayBanner ? 'block' : 'none';
 
+  const button = {
+    id: announcement.buttonId
```

Comment:
I just copied this from https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx#L191-L211 which is how buttons were defined in `SpecialAnnouncementActionBlock`. It could be undefined if a person on marketing doesn't define a buttonId in Announcements.json. I could rename the default to `marketing-announcement-banner-button`, if that makes more sense to you?

Reply:
Ah, I see.  Yeah, if it's not important that the name matches the `SpecialAnnouncementActionBlock`, then I would say use a name other than `special-announcement-btn-1` (to make it clear that these two things are not connected).